app/components/loading-indicator/template.hbs

```diff
@@ -0,0 +1 @@
+<div class='ball-scale ball-light LoadingIndicator'><div></div></div>
```

Comment:
Some extra closing div tags here

Reply:
Don't think so. Because of how our loading indicator works, it expects a div inside `.ball-scale.ball-light`

Reply:
As in, that's how osf-style has this working, for whatever reason. Once (if) we integrate osf-styles into our own, we should be able to move this stuff to just the component.

Reply:
👍 fair. I misread that--got a little confused without the indentation. It looked like 3 closing div tags the first time I read it.